tests/functional/spec/exit.spec.js

```diff
@@ -1,33 +0,0 @@
-import CensusThankYouPage from "../base_pages/census-thank-you.page.js";
```

Comment:
How come this file was removed?

Reply:
I was testing something and pushed it up with other changes, reverted it now.